Automattic/wp-calypso
# Remove support for Directly RTM support channel from Calypso everywhere (including the Help Center)

## Summary

As of September we no longer use Directly support - pcbce7-2tb-p2. For this reason we need to remove the Directly support features in Help Center and Calypso.

This removes all Directly calls and logic from help center and the data store. And removes all the redux state that manages Directly in Calypso. 

## Testing

1. Pull branch and `cd apps/editing-toolkit/ && yarn dev --sync` to sync ETK
2. Run `yarn start` from Calypso folder
3. Sandbox your Public API and your test site
4. Login with a user in the test group: Use `testing10percentid` in the Secret Store. That should enable the Help Center
5. Login to HappyChat in staging: `https://hud-staging.happychat.io/` and set yourself ready to accept chats.
6. Test the Help Center in Calypso, in a page like `http://calypso.localhost:3000/stats/day`. Try to start a chat.
7. Go to the Editor, and try to do the same.
8. Login with a free user, redo the previous tests. You should be given Forums as support.
9. Login with a user whose last 2 digits in the id are greater than 10 (so not in the test group, should see the FAB)
10. Test FAB to ensure no regression. Try to start a chat.
